### ISSUE TITLE
Recursive mode might be processing folders multiple times

### ISSUE THREAD

**Issue by lnicola** [alvistar/immich-albums#11]
This is a test with an empty directory, but on a real one it appeared to process even the parent multiple times

```
Processing folder: /tank/path/Folder

Album Folder does not exist
DRY RUN: Creating album Folder
DRY RUN: Assets ids: []
Processing folder: /tank/path/Folder/Subfolder

Album Subfolder does not exist
DRY RUN: Creating album Subfolder
DRY RUN: Assets ids: []
Processing folder: /tank/path/Folder/Subfolder

Album Subfolder does not exist
DRY RUN: Creating album Subfolder
DRY RUN: Assets ids: []
```

This might be because of the recursive call on https://github.com/alvistar/immich-albums/blob/main/src/immich_albums/im.py#L150-L154 (since `os.walk` is already recursive).

**Comment by nothing2obvi** [comment]
It's definitely doing this for me. It keeps running through the same folder over and over.

**Comment by alvistar** [comment]
Should be fixed  with #16

**Comment by alvistar** [comment]
Should be fixed  with pr  #16
